src/AuthenticatedAPIClient/axiosConfig.js

```diff
@@ -76,31 +76,52 @@ function applyAxiosInterceptors(authenticatedAPIClient) {
       authenticatedAPIClient.refreshAccessToken()
         .then(() => {
           queueRequests = false;
-          PubSub.publishSync(ACCESS_TOKEN_REFRESH);
+          PubSub.publishSync(ACCESS_TOKEN_REFRESH, { success: true });
         })
-        .catch(() => {
-          // TODO: We should give the client app an opportunity to
-          // take control here before logout/redirect to sign in.
-          authenticatedAPIClient.logout();
+        .catch((error) => {
+          if (authenticatedAPIClient.handleRefreshAccessTokenFailure) {
+            authenticatedAPIClient.handleRefreshAccessTokenFailure(error);
+          } else {
+            authenticatedAPIClient.logout();
```

Comment:
Can we add the following comment back?  You may have lost it during the test clean-up, and I think it helps clarify the README (which says that you will login, not logout).
```
// If no callback is supplied frontend-auth will (ultimately) redirect the user to login.
// The user is redirected to logout to ensure authentication clean-up, which in turn
// redirects to login.
```